src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -1062,4 +1086,22 @@ public void onBuildingUpgradeComplete(AbstractBuilding building, int level)
     {
         return ServerUtils.getPlayersFromUUID(this.world, this.getPermissions().getMessagePlayers());
     }
+
+    public void triggerAchievement(Achievement achievement)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)